testdata/p4_16_samples/parserUnroll/example1.p4

```diff
@@ -0,0 +1,105 @@
+#include "include/v1model.p4"
```

Comment:
why create a new directory inside p4_16_samples?
Perhaps you can instead name these files as parser-unroll-test1.p4, etc, and keep them in the same directory.